content/katas/fibonacci sequence/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Fibonacci sequence
+pre: "<b>1. </b>"
+weight: 5
+---
+
+
+ **What is fibonacci sequence?**
+
+> Fibonacci Sequence: is the series of numbers where the next number is found by adding up the n numbers before it.
```

Comment:
The next number is found by adding up the two numbers before it, not n numbers before it.